e2e/tests/plugins/notebook/tags.e2e.spec.js

```diff
@@ -137,10 +146,55 @@ test.describe('Tagging in Notebooks', () => {
         await expect(page.locator('[aria-label="Search Result"]')).not.toContainText("Driving");
     });
     test('Tags persist across reload', async ({ page }) => {
-        await createNotebookEntryAndTags(page);
+        //Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Create a clock object we can navigate to
+        await page.click('button:has-text("Create")');
+
+        // Click Clock
+        await page.click('text=Clock');
+        // Click button:has-text("OK")
+        await Promise.all([
+            page.waitForNavigation(),
+            page.locator('[name="mctForm"] >> text=My Items').click(),
+            page.locator('button:has-text("OK")').click()
+        ]);
+
+        await page.click('.c-disclosure-triangle');
```

Comment:
what element is this?

Reply:
The little arrow/triangle next to the tree item that expands the tree to show objects.